core/object/script_language.h

```diff
@@ -305,7 +306,7 @@ class ScriptLanguage : public Object {
 	virtual Error open_in_external_editor(const Ref<Script> &p_script, int p_line, int p_col) { return ERR_UNAVAILABLE; }
 	virtual bool overrides_external_editor() { return false; }
 
-	/* Keep enum in Sync with:                               */
+	/* Keep enums in Sync with:                               */
```

Comment:
I would suggest removing the trailing `*/` since it's now out of alignment, and doesn't add much anyway.
Probably best would be to change the comment style to `//` (and while at it change "Sync" to "sync").